templates/repo/branch/list.tmpl

```diff
@@ -19,12 +19,12 @@
 					<tbody>
 						<tr>
 							<td>
-								{{if .DefaultBranchBranch.IsProtected}}
-									{{svg "octicon-shield-lock"}}
-								{{end}}
 								<div class="gt-df gt-ac">
+									{{if .DefaultBranchBranch.IsProtected}}
+										<span class="flex-text-inline gt-pr-2">{{svg "octicon-shield-lock"}}</span>
```

Comment:
The parent already has "align-items: center", this `flex-text-inline` is a no op for a single SVG element.